src/packages/excalidraw/index.tsx

```diff
@@ -37,6 +37,8 @@ const ExcalidrawBase = (props: ExcalidrawProps) => {
     autoFocus = false,
     generateIdForFile,
     onLinkOpen,
+    registerComment,
+    user,
```

Comment:
**suggestion**: Can you define types for these two in `ExcalidrawProps` ?

Reply:
I forgot to push the latest code, will update it once I get back!

Reply:
My bad, I only pushed the code which is inside /src/packages/ folder. will push it once I get back!

Reply:
done